### PR TITLE
 feat(Banners): Add config to show print banner for uk election

### PR DESCRIPTION
**Pull request by thegreatercurve**
Adds UK election top banner.

[ticket](https://trello.com/c/Xtzt9Khi/1711-uk-election-banner)

### CODE REVIEW

File: templates/partials/top/print-banner-uk-election.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{> n-messaging-client/templates/components/o-message
+	theme='marketing__anon-subscribe'
+	contentTitle='Follow the fallout from the Election with the FT.'
+	buttonUrl='https://subs.ft.com/spa3_bc5?segmentId=79c8c0c5-1686-7f28-3aeb-c14688d59749'
+	buttonLabel='Choose your FT trial'
+	closeButton=false
+	customClass='n-messaging-client-alert-banner--marketing__anon-subscribe--dark'
+	customDataTrackableBanner='marketing-promo:header'
+	customDataTrackableActions='marketing-promo:box'
+	customDataTrackableButton='marketing-promo:link'
```
Comment (umbobabo) [inline templates/partials/top/print-banner-uk-election.html]: Same question as here https://github.com/Financial-Times/n-messaging-client/pull/219#discussion_r354745880.

Comment (thegreatercurve) [reply templates/partials/top/print-banner-uk-election.html]: Same answer as [here](https://github.com/Financial-Times/n-messaging-client/pull/219#discussion_r354768176) 😆 

I've taken them out though for the moment, and left a message for Aislinn in the ticket to see what she says.

File: manifest.js [hunk 2]
```diff
@@ -29,6 +29,9 @@ module.exports = {
 	anonSubscribeNow: {
 		path: 'top/anon-subscribe-now'
 	},
+	printBannerUkElection: {
```
Comment (alexnaish) [inline manifest.js]: Just going to pick hairs on the name of the banner - its not a print specific message so shouldn't have print in the name

Comment (thegreatercurve) [reply manifest.js]: Happy to change. `ukElection` instead?

Comment (alexnaish) [reply manifest.js]: Sounds good!

**Pull request merged**